### ISSUE TITLE
Screenreader Issue - full aria-describedby not being read

### ISSUE THREAD

**Issue by James-Green** [w3c/wai-quick-start#294]
In Developing, VO does not read the numbers that are included in the aria-describedby for Phone input under “help users avoid and correct mistakes”.  It only reads “for example”
![screen shot 2015-09-30 at 10 17 06 pm](https://cloud.githubusercontent.com/assets/13589074/10212151/ba131b08-67c0-11e5-9c6c-b0fa55cb8c05.png)

iOS 8 Voiceover with Mobile Safari

**Comment by iadawn** [comment]
It seems that if VO identifies the item as a phone number based on it's pattern, then it does not read it as part of the `aria-describedby`. If I swipe through the items, the phone number is read as a separate item.

This appears to be a 'feature' with VO on mobile. VO on desktop reads this as would be expected.

**Comment by iadawn** [comment]
This is an issue with iOS VO. Closing issue.

If you feel we should investigate this further, please do reopen.
